### ISSUE TITLE
AlDente on Macbook Air M3

### ISSUE THREAD

**Issue by TaylorSwift1803** [AppHouseKitchen/AlDente-Charge-Limiter#1238]
Hi, I want to ask that whether AlDente supports Macbook Air M3 base or not? Because on the supported devices I can’t see Macbook Air M3. Thank you in advance!

**Comment by vrognas** [comment]
Yes the MacBook Air M3 is supported, it is clearly stated under "Is my MacBook Model supported?" on https://apphousekitchen.com/faq/
